.tool-versions

```diff
@@ -11,3 +11,4 @@ python 3.12.9
 shfmt 3.11.0
```

Comment:
I don't see anywhere where we're actually using python. Do you remember what we're using it for?

Reply:
I think it was being used for one of the elixir libraries for building a NIF.